dlt/extract/incremental/__init__.py

```diff
@@ -502,7 +502,13 @@ def __call__(self, rows: TDataItems, meta: Any = None) -> Optional[TDataItems]:
             # add directly computed hashes
             unique_hashes.update(transformer.unique_hashes)
             self._cached_state["unique_hashes"] = list(unique_hashes)
-
+            dedup_count = len(self._cached_state["unique_hashes"])
+            DEDUP_WARNING_THRESHOLD = 200
+            if dedup_count > DEDUP_WARNING_THRESHOLD:
```

Comment:
this will flood user with messages. please compare the number of hashes before `unique_hashes.update(transformer.unique_hashes)` and after to see if threshold is crossed. so you display that only once

Reply:
Resolved in dd66d42